file_manage.rb

```diff
@@ -0,0 +1,15 @@
+afile = File.open("test.rb" , "r+")
+if afile
+  afile.syswrite('hello GAMMASTACK')
```

Comment:
Either use `' '` or use `" "`. Don't use both untill and unless its necessary. We generally prefer `' '`.